rasa/shared/core/training_data/story_reader/yaml_story_reader.py

```diff
@@ -46,6 +49,8 @@
 
 
 CORE_SCHEMA_FILE = "utils/schemas/stories.yml"
+DEFAULT_VALUE_TEXT_SLOTS = "filled"
```

Comment:
So, these two stories would still be equivalent?

```yaml
stories:
- story: Pasta 1
  steps:
  - intent: order_pasta
  - slot_was_set:
    - pasta_type
- story: Pasta 2
  steps:
  - intent: order_pasta
  - slot_was_set:
    - pasta_type: "filled"
```
😬 😬 😬

Reply:
Yes 👍  It doesn't matter for `text` slots as the ML model only looks **if** the slot was filled. So the actual text doesn't matter at all.

Reply:
Aha ok - it never looks at the value... Makes sense! (I missed that in the description 🤦 )

Reply:
Yeah. Not a huge fan of this "reference by name" only thing 🤷‍♂️